.semaphore/semaphore.yml

```diff
@@ -45,10 +45,167 @@ blocks:
         - name: AWS_REGION
           value: "us-west-2"
         - name: IMAGE_REPOSITORY
-          value: "confluentinc/boring-registry"
+          value: "docker/prod/confluentinc/boring-registry"
       jobs:
-        - name: "Bump version and Publish Docker image"
+        - name: "Bump version and prepare release"
           commands:
-            - curl -sfL https://goreleaser.com/static/run > /tmp/goreleaser && chmod +x /tmp/goreleaser && sudo mv /tmp/goreleaser /usr/local/bin/goreleaser
             - ./auto-release.sh
+
+  - name: "Prod Build AMD64"
```

Comment:
could you combine the dev and prod jobs to reduce duplication? It looks like the core differences at this point are the image path and the version determination. These should be easy to address with a conditional based upon the branch